modules/gcve-private-cloud/main.tf

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  region = join("-", slice(split("-", "${var.zone}"), 0, 2))
```

Comment:
nit: you're using string interpolation syntax for var.zone where it's not required.

also, alternate approach could be
`regex("([a-z]*-[a-z]*[0-9]{1,2})-([a-z])", var.zone)[0]`